site/src/components/popover.md

```diff
@@ -71,3 +71,26 @@ Popover is left aligned by default, but also can be centered or right aligned re
 	<div class="popover-content border">Popover content.</div>
 </details>
 ```
+
+## Closing popover
+
+An additional way to collapse the popover is to add the `data-close` attribute to the link/button in the content.
```

Comment:
I think we should name scope this to popovers `data-popover-close`?

Reply:
Updated